shell/core/types.ts

```diff
@@ -234,11 +235,6 @@ export interface ProductOptions {
    */
   ifHaveType?: string | RegExp;
 
-  /**
-   * Hide the product if the type is present (opposite of ifHaveType)
-   */
-  ifNotHaveType?: string | RegExp;
```

Comment:
careful about this... probably is because it's missing a rebase or something. This was added quite recently. Should not be deleted